### PR TITLE
Fixed all references to the old docker-compose command to use newer 'docker compose'

### PR DESCRIPTION
**Pull request by rjurney**
#94 kept `dc up` from working. This PR fixes the issue.

### CODE REVIEW

**Comment by lmeyerov** [comment]
The are some automation scripts in this repo that launch images that likely don't have the `docker compose` plugin, just the `docker-compose` command, so we should probably wait on this PR as part of a broader upgrade? Or to land sooner, do some sort of availability check?

**Comment by rjurney** [comment]
I see now that this is only a partial fix for the `src/docker` folder and the rest of the repo has a lot of `docker-compose` instances. What if we did an environment / version specific alias in the build as part of a fix here when we do the entire repo, such as:

```bash
alias "docker-compose"="docker compose"
```

That way the old command will work if the build expects it, otherwise it won't? We could detect the docker-ce version.

**Comment by lmeyerov** [comment]
maybe reuse what we have elsewhere as a stop gap: https://github.com/graphistry/graphistry/blob/9a93d7c90346b8cf35e16468ce8774645e0b16dd/compose/etc/scripts/migrate.sh#L159

File: src/docker/dc [hunk 1]
```diff
@@ -4,5 +4,5 @@ set -ex
 
 DOCKER_BUILDKIT=1 \
 COMPOSE_DOCKER_CLI_BUILD=1 \
-    docker-compose $@
+    docker compose $@
```
Comment (lmeyerov) [inline src/docker/dc:7]: this conflicts with thomas's pr, which should be the one we land, right?

**Pull request closed without merging**